components/collection/drop/PaidGenerative.vue

```diff
@@ -102,42 +78,38 @@ const action = computed<AutoTeleportAction>(() => ({
   },
 }))
 
-const {
-  claimedNft,
-  mintedNftWithMetadata,
-  selectedImage,
-  tryCapture,
-  subscribeToMintedNft,
-  listMintedNft,
-} = useGenerativeDropMint()
-const { description, collectionName, nftCount } = useCollectionEntity()
-
+const { collectionName } = useCollectionEntity()
+const { amountToMint, previewItem } = storeToRefs(useDropStore())
+const { mintingSession, toMintNFTs, allocatedNFTs } = useDropMassMintRefs()
 useCursorDropEvents([isTransactionLoading, isLoading])
 
 const mintNft = async () => {
   try {
     isLoading.value = true
+    mintingSession.value.txHash = undefined
```

Comment:
this is why you are passing as prop isn't it?

so why is this ref coming from the composable to begin with?